core/src/main/java/io/fabric8/maven/core/config/ResourceConfig.java

```diff
@@ -92,7 +92,10 @@
     @Parameter
     private List<ServiceAccountConfig> serviceAccounts;
 
-    private List<IngressRule> ingressRules;
```

Comment:
I added this wrongfully in my previous PR here: https://github.com/fabric8io/fabric8-maven-plugin/pull/1631/commits/dc5b213995d55e7470e08d4f2a9cbf7879546c9a#diff-9238e96d57160d3b61190b609607b2d8R95

hence, removing it

Reply:
Remove import above too